File: nice.py
from random import randint

def rand():
   return randint(10000, 99999)
count = 0
while True:
   count+=1
   finder = rand()
   if finder == 69420 or finder == 42069:
      print(f"{finder} Found! - {count} times to find")
      break
   print(finder)

File: nice.swift
var count: Int = 0

func random() -> Int {
    return Int.random(in: 10000...99999)
}
repeat {
    count+=1
    let find: Int = random()
    if find == 42069 || find == 69420{
        print("\(find) Found! \(count) times to find.")
        break
    }
    print(find)
} while true
